ledger/participant-integration-api/src/main/scala/platform/indexer/parallel/ParallelIndexerFactory.scala

```diff
@@ -68,7 +68,16 @@ object ParallelIndexerFactory {
                   Executors.newFixedThreadPool(
                     1,
                     new ThreadFactoryBuilder().setNameFormat(s"ha-coordinator-%d").build,
-                  )
+                  ),
+                  throwable =>
+                    LoggingContext.newLoggingContext { implicit loggingContext =>
```

Comment:
Is this new logging context so that it's a "clean" log line? The enclosing scope already has an implicit `LoggingContext` value available.

Reply:
Thanks - reusing the embedding `LoggingContext` here and also by passing through the implicit to `AsyncSupport` and `DbDispatcher`.